### PR TITLE
Fatima: Wait task

### PR DESCRIPTION
**Pull request by FatimaFarah93**
PR is ready for review!

### CODE REVIEW

File: src/main/java/session17waitsandsync/examples/fatima/implicitWait.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.By;
+
+import java.util.concurrent.TimeUnit;
+
+
+public class implicitWait {
+    public static void main(String[] args) {
+        WebDriver driver = new ChromeDriver();
```
Comment (owynowyn) [inline src/main/java/session17waitsandsync/examples/fatima/implicitWait.java:13]: `driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));` to set an implicit wait, after the driver has been set up that is.

Comment (owynowyn) [reply src/main/java/session17waitsandsync/examples/fatima/implicitWait.java:13]: That will then implicitly wait every time you interact with the driver, so you won't need to do anything else after the above line of code has been added - https://www.selenium.dev/documentation/webdriver/waits/

**Review by owynowyn — CHANGES_REQUESTED** [review]
1 small change to add but it is pretty much there! 💯

**Comment by FatimaFarah93** [comment]
I've done that change now :)

**Review by owynowyn — APPROVED** [review]
Great work!

**Pull request merged**